stroom-util-shared/src/main/java/stroom/util/shared/ErrorMessage.java

```diff
@@ -17,13 +17,23 @@ public final class ErrorMessage {
     @JsonProperty
     private final String message;
 
+    @JsonProperty
+    private final String node;
+
     @JsonCreator
     public ErrorMessage(@JsonProperty("severity") final Severity severity,
-                        @JsonProperty("message") final String message) {
-        this.severity = severity;
+                        @JsonProperty("message") final String message,
+                        @JsonProperty("node") final String node) {
+        this.severity = severity == null ? Severity.ERROR : severity;
```

Comment:
Objects.requireNonNullElse(severity, Severity.ERROR)

Reply:
Doesn't this need to be `NullSafe.requireNonNullElse(severity, Severity.ERROR)` so it works with GWT?

Reply:
ah yes, good point, forgot it was shared